lib/widgets/notes_screen_container.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+class NoteScreenCont extends StatelessWidget {
+  String title;
```

Comment:
As this field isn't final it would mark it as @immutable. Add the final keyword here.